pages/_app.tsx

```diff
@@ -2,12 +2,23 @@ import "nav-frontend-lenker-style";
 import "nav-frontend-knapper-style";
 
 import {useEffect} from "react";
+import * as Sentry from "@sentry/browser";
+import {v4 as uuid} from "uuid";
 
 import {initAmplitude} from "../src/utils/amplitude";
 
 function App({Component, pageProps}) {
     useEffect(() => {
         initAmplitude();
+        if (
+            window?.location?.href.startsWith("https://www.nav.no/sosialhjelp")
```

Comment:
Litt hackete måte å sjekke om vi er i prod på. Jobber med å få på plass bygg ved deploy til miljø, og da kan vi injecte miljøspesifikke variabler i stedet for å sjekke på `window.location`.